apps/opik-backend/src/test/java/com/comet/opik/api/resources/utils/TestUtils.java

```diff
@@ -0,0 +1,14 @@
+package com.comet.opik.api.resources.utils;
+
+import lombok.experimental.UtilityClass;
+
+import java.net.URI;
+import java.util.UUID;
+
+@UtilityClass
+public class TestUtils {
```

Comment:
Minor: better use a more specific name for this class. Having said that, this is a nice change.